src/drivers/px4fmu/fmu.cpp

```diff
@@ -189,9 +235,8 @@ class PX4FMU : public device::CDev
 	unsigned	_pwm_alt_rate;
 	uint32_t	_pwm_alt_rate_channels;
 	unsigned	_current_update_rate;
-	int 		_task;
 	bool 		_run_as_task;
-	struct work_s	_work;
+	static struct work_s	_work;
```

Comment:
There's a dependency problem: we want to instantiate the object within the work queue, but we need the `work_s` item to kick off the work queue before that.